vendor/github.com/armPelionEdge/greasego/deps/include/version.h

```diff
@@ -26,4 +26,4 @@
 */
 
 #define GREASE_LIB_VERSION "0.1.2"
-#define GREASE_LIB_COMMIT "5b578dd"
+#define GREASE_LIB_COMMIT "6cf0b3e"
```

Comment:
Should this file be reverted?

Reply:
Yes, I'll revert this. Thanks for pointing this out.